### ISSUE TITLE
Was the new [WindowsOnly] tag approved?

### ISSUE THREAD

**Issue by BenTheElder** [kubernetes/kubernetes#74740]
There is now a `[WindowsOnly]` test tag from https://github.com/kubernetes/kubernetes/pull/72777

I remember a `[LinuxOnly]` tag being approved in SIG Architecture and SIG Testing, but I don't remember a `[WindowsOnly]` tag. 

If this was approved I think we need to broadcast it more widely as many existing test setups including  current conformance testing appear to now need to skip / exclude this tag.

cc @timothysc  @spiffxp @dims 

/area conformance
/sig architecture
/sig testing

**Comment by BenTheElder** [comment]
@kubernetes/cncf-conformance-wg 

bug may not be the appropriate type, but seemed like the closest issue template to start from initially.

**Comment by timothysc** [comment]
TL;DR nope, that was not something that I'm aware of.

**Comment by jchauncey** [comment]
cc @PatrickLang

**Comment by craiglpeters** [comment]
The email thread is in [kubernetes-sig-testing](https://groups.google.com/forum/#!search/%5BWindowsOnly%5D$20kubernetes/kubernetes-sig-testing/ii5584-Tkqk/R2VEmjkgEAAJ)

**Comment by BenTheElder** [comment]
the [last message](https://groups.google.com/d/msg/kubernetes-sig-testing/ii5584-Tkqk/-86aO7t5FQAJ) in that thread says:

>  If there is a test case that's been added or changed to only work on Windows, it will be tagged [SIG-Windows], and will not be tagged as [Conformance] or [NodeConformance] because that could break existing test jobs and the conformance testing tool.

That's not the case here.

**Comment by dims** [comment]
@craiglpeters we should file a PR marked 1.14 and `/hold` to yank that tag, just so we don't forget. Of course if the decision changes in the email thread, we can revisit the PR. ok?

**Comment by craiglpeters** [comment]
@dims makes sense to me

**Comment by bgrant0607** [comment]
Catching up. Quick comment before I read the threads:

Documented kinds of tests are currently here:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests

WindowsOnly had been hypothesized, but not yet added to that list.

**Comment by bgrant0607** [comment]
Also, are there already tests tagged as SIG Windows?

**Comment by bgrant0607** [comment]
On conformance tests:

Windows isn't part of conformance yet, so no conformance test should have this tag.

And no conformance test should contain any Skip directives. They should die in that case:
https://github.com/kubernetes/kubernetes/issues/74432

**Comment by bgrant0607** [comment]
I looked at #72777 and don't understand why the Windows image test wasn't written as a separate test. To avoid either refactoring or copy/pasting? It looks like a pretty obscure approach -- auto-skipping individual test cases that contain a particular tag?

**Comment by bclau** [comment]
@bgrant0607 Pretty much. But it is no longer the case, since all those "should pull image from X" test cases have been refactored. (look at https://github.com/kubernetes/kubernetes/pull/74655 to see how it looks now)

**Comment by bgrant0607** [comment]
@bclau Yes, that's better, thanks.

To answer my own question regarding `[sig-windows]`, here's an example:

https://github.com/kubernetes/kubernetes/blob/f7eb5769c2dc4bc701802da639451d0f6cd56a18/test/e2e/windows/networking.go#L29

which also invokes `framework.SkipUnlessNodeOSDistroIs("windows")` inside the test.

So we're agreed that we don't need `[WindowsOnly]` yet?

**Comment by neolit123** [comment]
https://github.com/kubernetes/kubernetes/pull/74655
now removes all instances of the [WindowsOnly] tag.
